tagged-manifest.xml

```diff
@@ -3,5 +3,5 @@
   <!-- incremental version bump: droid-src-sony-aosp-11/1.9.3 -->
   <remote name="hybris" fetch="https://github.com/mer-hybris"/>
   <include name="tagged-localbuild.xml"/>
-  <project name="droid-src-sony" path="rpm" remote="hybris" revision="870d56405517e6217bfef5894ce1dacf38b7d95e"/>
+  <project name="droid-src-sony" path="rpm" remote="hybris" revision="0d2fda3c8903086c41f33d020e0c2700e70d1aba"/>
```

Comment:
Please change this PR to draft to prevent accidental merging, before the depending [PR](https://github.com/mer-hybris/droid-src-sony/pull/27/commits) is merged:) This hash will need updating to a merge commit.